.codecov.yml

```diff
@@ -195,6 +203,10 @@ coverage:
         target: 75
         flags:
         - cert_manager
+      gRPC_Check:
+        target: 75
+        flags:
+        - grpc_check
```

Comment:
nit - can you move it between `gnatsd_streaming` and `jfrog_platform`? (no need to move `cert_manager`, it is being migrated to -core so it will be removed very soon)